modules/tensor_mechanics/include/auxkernels/CopyValueAux.h

```diff
@@ -0,0 +1,25 @@
+/*
+AuxKernel of Passing Variable
+*/
+
+#pragma once
+
+#include "AuxKernel.h"
+
+class CopyValueAux : public AuxKernel
```

Comment:
use the SelfAux. No need for a copyValueAux

Reply:
(so delete this auxkernel please)

Reply:
SelfAux seems to work between AuxVariable to AuxVariable, however here I would like to copy Variable to AuxVariable, is there a workaround to do this?

Reply:
Can you try again?
You have two options, projectionAux or a new copyValueAux (I ended up stealing the name to rename SelfAix)

Reply:
> Can you try again?
> You have two options, projectionAux or a new copyValueAux (I ended up stealing the name to rename SelfAix)

This turns out to be working properly! Thanks, I have deleted this custom one.